device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnection.java

```diff
@@ -396,23 +333,21 @@ private void executorServicesCleanup() throws TransportException
 
     /**
      * Private helper for close.
-     * Closes the AmqpsSessionManager, the connection and stops the Proton reactor.
+     * Closes the AmqpsIotHubConnection, the connection and stops the Proton reactor.
      */
     private void closeAsync()
     {
-        this.log.trace("CloseAsync called on amqp connection");
-        // Codes_SRS_AMQPSIOTHUBCONNECTION_15_013: [The function shall closeNow the AmqpsSessionManager and the AMQP connection.]
-        if (this.amqpsSessionManager != null)
-        {
-            this.amqpsSessionManager.closeNow();
-        }
+        this.log.debug("Closing amqp connection...");
```

Comment:
is this invoked only on explicit deviceclient.closeAsync() call, or during reconnect as well?